Snakefile

```diff
@@ -129,16 +96,13 @@ def make_final_input(wildcards):
 
 # A rule to define all the expected outputs from all pathway reconstruction
 # algorithms run on all datasets for all arguments
+# TODO: do we need this rule? Does it kick off all other rules?
```

Comment:
Yes, that's what it does.  By default Snakemake will run the first rule.  It is possible to override that by specifying a different rule for the command line.

To make this better match the current behavior, we could rename the rule `all`.  This rule now does more than just reconstruct pathways.